### PR TITLE
chore: refine logging in pkg/ptrace

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

This PR is waiting for merging https://github.com/chaos-mesh/chaos-mesh/pull/2964

### What problem does this PR solve?

close  https://github.com/chaos-mesh/chaos-mesh/issues/2885

### What's changed and how it works?

- require logger when using `ptrace.Trace`

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [x] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL

/cc @Andrewmatilde @iguoyr

File: pkg/time/fake_image_linux_amd64.go [hunk 1]
```diff
@@ -37,7 +38,8 @@ func (it *FakeImage) AttachToProcess(pid int, variables map[string]uint64) error
 		runtime.UnlockOSThread()
 	}()
 
-	program, err := ptrace.Trace(pid)
+	// TODO: Waiting for #2889, use logger provided by FakeImage
```
Comment (iguoyr) [inline pkg/time/fake_image_linux_amd64.go:41]: @STRRL we could use logger provided by FakeImage now, according to #2889 is merged.

Comment (Andrewmatilde) [reply pkg/time/fake_image_linux_amd64.go:41]: I agree with it.

Comment (STRRL) [reply pkg/time/fake_image_linux_amd64.go]: Updated! PTAL!

**Review by iguoyr — APPROVED** [review]
LGTM!

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Pull request merged**